packages/rancher-monitoring/rancher-monitoring.patch

```diff
@@ -1657,6 +1668,60 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/val
 +    - effect: "NoSchedule"
 +      operator: "Exists"
 +
++## rke2 PushProx Monitoring
++## ref: https://github.com/rancher/charts/tree/master/packages/rancher-pushprox
++##
++rke2ControllerManager:
++  enabled: false
++  metricsPort: 10252
++  component: kube-controller-manager
++  clients:
++    port: 10011
++    useLocalhost: true
++    nodeSelector:
++      node-role.kubernetes.io/master: "true"
++    tolerations:
++      - effect: "NoExecute"
++        operator: "Exists"
++      - effect: "NoSchedule"
++        operator: "Exists"
++
++rke2Scheduler:
++  enabled: false
++  metricsPort: 10251
++  component: kube-scheduler
++  clients:
++    port: 10012
++    useLocalhost: true
++    nodeSelector:
++      node-role.kubernetes.io/master: "true"
++    tolerations:
++      - effect: "NoExecute"
++        operator: "Exists"
++      - effect: "NoSchedule"
++        operator: "Exists"
++
++rke2Proxy:
++  enabled: false
++  metricsPort: 10249
++  component: kube-proxy
++  clients:
++    port: 10013
++    useLocalhost: true
```

Comment:
Do we not need the tolerations settings here ?

Reply:
In my testing it didnt need any, but added anyway just in case

Reply:
@brandond or @cjellick do we know which parts we do or don't want tolerations on ?

Reply:
Are we talking about the bootstrap flag on the HelmChart that adds tolerations? If so, I think we should only put those on things that need to run as part of cluster bootstrap. Other stuff like monitoring can wait until after the core functionality is up.

Reply:
Oh, I see. The these are just to let monitoring run on masters or nodes that have been tainted NoExecute/NoSchedule? Seems fine to me.

Reply:
Ok, if we are good with that. I was more concerned about users being able to edit this setting via values.